core/admin/mailu/models.py

```diff
@@ -126,8 +139,8 @@ class Relay(Base):
 
     __tablename__ = "relay"
 
```

Comment:
Not sure this is necessarily to be handled as a mail domain, as it is merely  hostname.

Reply:
Probably not, so reseting it in c40e255f3b513fb7846d4a90e5057c69c4b90c72